app/controllers/books_controller.rb

```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
 
   # GET /books or /books.json
   def index
-    @books = Book.all
+    @books = Book.order('id').page(params[:page]).per(1)
```

Comment:
```suggestion
    @books = Book.order(:id).page(params[:page]).per(1)
```

シンボルを指定することが多いですね。
あと、`per(1)`は動作確認ではやむなしかもしれませんが、現実的には滅多に設定しない数字なので、課題提出時には自然な形に直してもらいたいですー。

Reply:
ありがとうございます🙏